tests/e2e/scenarios/test_admin_api.py

```
@@ -1,153 +1,245 @@
 
-from helpers import AUTH_TOKEN
+from helpers import REBORN_V2_AUTH_TOKEN
+from reborn_webui_harness import (
+    reborn_bearer_headers,
+    reborn_v2_server,  # noqa: F401 - imported fixture
+)
+
+ADMIN_BASE = "/api/webchat/v2/admin"
 
 
 @pytest.fixture()
```

Ruff (0.15.20)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)